scripts/freeze_backref_cost_aligned_null_stage_attribution_v2_phase1.py

```
@@ -0,0 +1,1080 @@
+            ROOT / "scripts/backref_cost_aligned_null_stage_attribution_v2_identity.py"),
+    }
+
+
+def write_json(path: Path, value: object) -> None:
+    path.parent.mkdir(parents=True, exist_ok=True)
+    path.write_text(json.dumps(value, indent=2, sort_keys=True) + "\n",
+                    encoding="utf-8")
+
+
+def run(argv: list[str]) -> str:
+    result = subprocess.run(argv, check=False, stdout=subprocess.PIPE,
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)